releases.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Releases
+layout: default
+---
+
+<table  class="table-striped">
```

Comment:
Similar to the News page, maybe add `table-hover`?

Reply:
will change..